README.md

```diff
@@ -111,3 +111,51 @@ Example:
 You will be prompted to confirm the deployment. When it finishes, you will be able to access the
 UI under http://PROJECT.appspot.com and the API under http://api.PROJECT.appspot.com.
 
+## git-secrets
+
+### Setup
+
+Download the git-secrets tool.
+If you have not yet run the setup environment script, run:
+```Shell
+./setup_env
```

Comment:
I think several of these are missing `tools` and `.sh`. This should be `tools/setup_env.sh` right?